transports/Dockerfile

```diff
@@ -36,11 +36,15 @@ ARG ENV_PATH
 ARG PORT
 ARG POOL_SIZE
 ARG DROP_EXCESS_REQUESTS
+ARG PLUGINS
+ARG MAXIM_LOGGER_ID
```

Comment:
we use `MAXIM_API_KEY` and `MAXIM_LOG_REPO_ID` as standard env variables

Reply:
yes okay. @danpiths said to take MAXIM_LOG_REPO_ID from args and ask the user to set MAXIM_API_KEY in the env beforehand